tests/models/test_restore.py

```diff
@@ -73,6 +73,18 @@ def test_model_properties_resume_from_checkpoint(enable_pl_optimizer, tmpdir):
     trainer.fit(model)
 
 
+def test_try_resume_from_non_existing_checkpoint(tmpdir):
+    """ Test that trying to resume from non-existing `resume_from_checkpoint` fail without error."""
+    model = EvalModelTemplate()
+    checkpoint_callback = ModelCheckpoint(dirpath=tmpdir, monitor="early_stop_on", save_last=True)
+    trainer = Trainer(default_root_dir=tmpdir, max_epochs=2, logger=False, checkpoint_callback=checkpoint_callback)
```

Comment:
```suggestion
    trainer = Trainer(default_root_dir=tmpdir, max_epochs=1, logger=False, checkpoint_callback=checkpoint_callback)
```
one epoch is enough, right?

Reply:
also, limit the batches, use `callbacks=[checkpoint_callbacks]` and use `BoringModel`.

Reply:
You are right, this PR is made 3 months ago, so do not based on new `callbacks` & `BoringModel`.  
I will check them and fix it!

Reply:
Fixed.